common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -44,4 +44,15 @@ trait ABTestSwitches {
     sellByDate = new LocalDate(2016, 12, 8), // Thursday 8th December
     exposeClientSide = true
   )
+
+  Switch(
+    ABTests,
+    "ab-au-memb-engagement-msg-copy-test-8",
```

Comment:
Heck, I think this might have been wrong @JustinPinner - it looks like the value should have been `ab-au-memb-engagement-msg-copy-test8`, without the dash before the 8?

The code that matches the Scala switch with the AB test id seems to require no dashes before numerals, and if they don't match, it silently deactivates the test. So as far as I can see, the AB test on the Australia edition has not been active - only the control would have been shown. No variants will have been displayed to users.

I tried this, and as you can see, the banner message doesn't match the selected variant:

![image](https://cloud.githubusercontent.com/assets/52038/21398470/537735b0-c79f-11e6-8759-f6e21e1ba2d5.png)